Components/Checks/CheckDnnVersion.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace DNN.Modules.SecurityAnalyzer.Components.Checks
+{
+    using DotNetNuke.Application;
+
+    public class CheckDnnVersion : IAuditCheck
+    {
+        public string Id => "CheckDnnVersion";
+
+        public bool LazyLoad => false;
+
+        public CheckResult Execute()
+        {
+            if (DotNetNukeContext.Current.Application.Version.Major < 9)
+            {
+                return new CheckResult(SeverityEnum.Failure, Id);
+            }
+
+            return new CheckResult(SeverityEnum.Warning, Id);
```

Comment:
Was unable to make this note in the resx as I couldn't find the version.  For a warning result, I would propose the test.

You are running DNN 9.x or later, this module is no longer being maintained and should be uninstalled.  You can use the Security Analyzer included in the PersonaBar for security concerns with this version of DNN.